docs/admin/deployment.rst

```diff
@@ -199,6 +202,12 @@ you create:
    Timezone for the dynos that will run the app. Pontoon operates in UTC, so set
    this to ``UTC``.
 
+``VCS_SYNC_NAME``
+  Optional. Default committer's name that Sync uses when a changeset doesn't have authors.
```

Comment:
We actually always use this committer (we only change author). So I'd change this (and similarly below) to:
```
Optional. Default committer's name used when committing translations to version control system.
```